src/views/CoCurricularTranscript/Components/Experience/Experience.module.scss

```
@@ -31,8 +31,29 @@
   }
 }
 
+.experience_transcript_activities_empty_titles {
+  display: grid;
+  grid-template-columns: 5% 70% 25%;
+  grid-template-rows: auto auto;
+  justify-items: stretch;
+  position: relative;
+  top: -20px;
+
+  .organization_role {
+    grid-column: 2;
+    grid-row: 1;
+    text-align: left;
+  }
+
+  .date {
+    grid-column: 3;
+    grid-row: 1;
+    text-align: left;
+  }
+}
+
```

It's not ideal to duplicate the whole styles, just to add the `position: relative` and `top: -20px`. It would be better to keep those in a separate class and simply apply both classes when needed:
```suggestion
.empty_title {
  position: relative;
  top: -20px;
}

```